layouts/account.tsx

```diff
@@ -41,13 +45,39 @@ type TabTypeEnum = "delegating" | "orchestrating" | "history";
 
 const ACCOUNT_VIEWS: TabTypeEnum[] = ["delegating", "orchestrating", "history"];
 
-const AccountLayout = ({
-  account,
-  sortedOrchestrators,
-}: {
-  account?: AccountQueryResult["data"] | null;
-  sortedOrchestrators: OrchestratorsSortedQueryResult["data"];
-}) => {
+const AccountLayout = () => {
+  /* PART OF https://github.com/livepeer/explorer/pull/427 - REMOVE ONCE SERVER-SIDE ISSUE IS FIXED */
```

Comment:
Let's mark this as `TODO: REMOVE ONCE SERVER-SIDE ISSUE IS FIXED` as that shows up in your editor under todos.